octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java

```diff
@@ -167,24 +191,19 @@ private void drawEntityFields(Set<String> shownFields, EntityModelWrapper entity
                 columnComposite = sectionClientRight;
             }
 
-            if (!fieldName.equals(EntityFieldsConstants.FIELD_DESCRIPTION)
-                    && !fieldName.equals(EntityFieldsConstants.FIELD_NAME)
-                    && !fieldName.equals(EntityFieldsConstants.FIELD_PHASE)) {
```

Comment:
@atothhpe Since you removed this condition, it means that we are treating the 3 fields as the rest of them?

Reply:
So this draws the fields onto the detail tab. But the fields that you can select come from the dropdown and the default fields. Since in the field select dropdown, these fields are removed, they can't end up being here.